### DISCUSSION TITLE
我的第一辆电动自行车 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#18] [Announcements]
# 我的第一辆电动自行车 - forecho's Blog

引言
去年年底买了辆电动自行车，到现在已经有半年多了，今天给大家分享一下我的购买经验和使用体验。
买前指南
需求和预算
买前先了解自己的需求和预算。
我的需求主要就是上下班通勤使用，一趟 3 公里。刚开始对电池、预算也没啥概念。看了几个测评，逛了几个实体店才有了一点了解。
电池太小，跑的就不远，充电要频繁一点。刚开始打算买个下面这种电动自行车就行的，便宜，小巧，停车方便。但是电池容量不高，「驾驶」体验也不是很好，后面只适合坐小朋友，成人坐着比较难受。

https://blog.forecho.com/my-first-electric-bike.html

**Comment by Cathay-Chen** [comment]
不错不错

**Comment by ting-talk** [comment]
短途骑小毛驴，最快乐了。
